### DISCUSSION TITLE
Initiate search of site without leaving page

### DISCUSSION THREAD

**Discussion by huumn** [stackernews/stacker.news#3009] [Ideas]
I often find myself wanting to supply links and references but don't want to leave the page so I don't.

I'm not sure how this should work exactly but afaik there are a few options:

1. inline text command (kind of like when you type `#` on github)
2. some kind of keyboard shortcut that pops up a modal

Ultimately, (1) will be less context rich but (2) will be heavier UI/UX wise so there are tradeoffs.

**Comment by SatsAllDay** [comment]
I don't necessarily want to work on this, but reading the description, I'm not sure I really understand what the issue is for, exactly? You want to supply links and references to other SN items while creating a reply on an item? Is that the use case? Or something else entirely?

**Comment by huumn** [comment]
> You want to supply links and references to other SN items while creating a reply on an item?

Basically. In any textarea I want to be able search for other SN items with minimal friction. This has a bonus of helping resurface old stuff too.

Case study: https://stacker.news/items/272853

Originally I was too lazy to go search for the old post.

This kind of thing happens a lot. Someone asks about something that has been discussed or posted before, or I'm making a point that touches on some earlier content.
